frontend/javascripts/oxalis/model.js

```diff
@@ -90,13 +95,67 @@ export class OxalisModel {
     );
   }
 
-  // Todo: Make return type optional
-  getSegmentationLayer(): DataLayer {
-    return _.find(
-      this.dataLayers,
-      dataLayer =>
-        getLayerByName(Store.getState().dataset, dataLayer.name).category === "segmentation",
+  getSegmentationLayers(): Array<DataLayer> {
+    return Object.keys(this.dataLayers)
+      .map(k => this.dataLayers[k])
```

Comment:
Is there a benefit of this to using `Object.values`? Flow?

Reply:
Yeah, only flow :/